src/store/tests/test-data/cat-b/test-data.cat-b.selector.ts

```diff
@@ -0,0 +1,82 @@
+import { get } from 'lodash';
+import { CompetencyOutcome } from 'src/app/shared/models/competency-outcome';
+import { CatBUniqueTypes } from '@dvsa/mes-test-schema/categories/B';
+import { QuestionProvider } from 'src/app/providers/question/question';
+import { TestCategory } from '@dvsa/mes-test-schema/category-definitions/common/test-category';
+import { VehicleChecksQuestion } from 'src/app/providers/question/vehicle-checks-question.model';
+import { Competencies, LegalRequirements } from '../test-data.constants';
+
+export const getDrivingFaultCount = (
+  data: CatBUniqueTypes.TestData, competency: Competencies,
+) => data.drivingFaults[competency];
+
+export const getManoeuvres = (data: CatBUniqueTypes.TestData): CatBUniqueTypes.Manoeuvres => data.manoeuvres;
+
+// TODO - We should pass a Manoeuvre object here instead of TestData
```

Comment:
is there a case for getting this comment addressed here?

Reply:
i don't think this is an issue so have removed the comment. changing the selector would mean updating all usages and I don't see the benefit / problem